source/documentation/tdb2/__index.md

```diff
@@ -3,11 +3,10 @@ title: TDB2
 slug: index
 ---
 
-TDB2 is a component of
-[Apache Jena](http://jena.apache.org) for RDF storage
-and query.  It support the full range of Jena APIs.
-TDB2 can be used as a high performance RDF store on a single machine.
-TDB2 can be used with Apache Jena Fuseki.
+TDB2 is a component of [Apache Jena](https://jena.apache.org) for RDF storage
```

Comment:
Thanks! That saves 1 hop as it won't have the HTTP redirect. While harmless, when running scanners like the W3C link checker or other analyzers that resolve links, having less redirects helps!